sw/device/lib/dif/dif_hmac.c

```diff
@@ -0,0 +1,221 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_hmac.h"
+
+#include "hmac_regs.h"  // Generated.
+#include "sw/device/lib/base/memory.h"
+
+#define HMAC_FIFO_MAX (HMAC_MSG_FIFO_SIZE_BYTES / 32)
```

Comment:
Assuming that you want the size in words here, shouldn't this be `HMAC_MSG_FIFO_SIZE_BYTES / 4` since `HMAC_MSG_FIFO_SIZE_BYTES` is in bytes? Actually, `hmac_regs.h` already defines `HMAC_MSG_FIFO_SIZE_WORDS` so it may be a better idea to use it directly.

Reply:
What a mess, yes I should be using `HMAC_MSG_FIFO_SIZE_WORDS`, thank you.